tests/functional/raw-node/test/GCP/object/get.js

```diff
@@ -0,0 +1,142 @@
+const assert = require('assert');
+const async = require('async');
+const { GCP } = require('../../../../../../lib/data/external/GCP');
+const { makeGcpRequest } = require('../../../utils/makeRequest');
+const { gcpRequestRetry } = require('../../../utils/gcpUtils');
+const { getRealAwsConfig } =
+    require('../../../../aws-node-sdk/test/support/awsConfig');
+
+const credentialOne = 'gcpbackend';
+
+describe('GCP: GET Object', function testSuite() {
+    this.timeout(30000);
+    const config = getRealAwsConfig(credentialOne);
+    const gcpClient = new GCP(config);
+
+    describe('with existing object in bucket', () => {
+        beforeEach(function beforeFn(done) {
+            this.currentTest.bucketName = `somebucket-${Date.now()}`;
+            this.currentTest.key = `somekey-${Date.now()}`;
+            gcpRequestRetry({
+                method: 'PUT',
+                bucket: this.currentTest.bucketName,
+                authCredentials: config.credentials,
+            }, 0, err => {
+                if (err) {
+                    process.stdout.write(`err in creating bucket ${err}\n`);
+                } else {
+                    process.stdout.write('Created bucket\n');
```

Comment:
In this repo I see tests both with explicit steps written to stdout and tests without them. I am in favor of removing stdout messages like this since the information can be assumed when no error occurred. What do you think?

Reply:
I agree. I had them for debugging; will clean up the stdout messages